prefect_census/__init__.py

```diff
@@ -1,4 +1,7 @@
 from . import _version
-from .blocks import CensusBlock  # noqa
+from .client import CensusClient  # noqa
+from .credentials import CensusCredentials  # noqa
+from .runs import get_census_sync_run_info  # noqa
+from .syncs import trigger_census_sync  # noqa
```

Comment:
I think we should only expose blocks on top level.
```suggestion
from .credentials import CensusCredentials  # noqa
```

Reply:
yeah that makes sense